docs/llm-hosting/bench/spectest.py

```
@@ -0,0 +1,44 @@
+    d = json.load(r)
+el = time.perf_counter() - t0
+u = d["usage"]
+print(f"prompt={u['prompt_tokens']} gen={u['completion_tokens']} wall={el:.1f}s "
+      f"TG={u['completion_tokens']/el:.2f} tok/s")
+
+# The throughput number is only meaningful if the model actually quoted back the
+# source. Compare against the handler_0..12 text; ignore_eos means output can run
+# past it, so check prefix agreement rather than equality.
+expect = "\n".join(CODE.split("\n\n")[:13]).strip()
+got = d["choices"][0]["text"].strip()
+n = next((i for i, (a, b) in enumerate(zip(expect, got)) if a != b), min(len(expect), len(got)))
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)